docs/src/main/sphinx/installation/rpm.rst

```diff
@@ -0,0 +1,91 @@
+================
+RPM package
+================
+
+Users can install Trino using the RPM Package Manager (RPM) on Linux
+distributions that support RPM.
+
+The RPM archive contains the application, all plugins, the necessary default
+configuration files, default setups, and integration with the operating system
+to start as a service.
+
+Installing Trino
+----------------
+
+Download the Trino server RPM package :maven_download:`server-rpm`. Use the
+``rpm`` command to install the package:
+
+.. code-block:: text
+
+    rpm -i trino-server-rpm-*.rpm --nodeps
+
+Installing the :ref:`required Java and Python setup <requirements>` is typically
+managed separately.
+
+Service script
+--------------
+
+The RPM installation deploys a service script configured with ``chkconfig`` so
+that the service can be started automatically on operating system boot. After
+installation, you can manage the Trino server with the ``service`` command:
+
+.. code-block:: text
+
+    service trino [start|stop|restart|status]
+
+* ``start`` Starts the server as a daemon and returns the process ID.
```

Comment:
do we want to add a dash like so ?

```suggestion
* ``start`` - starts the server as a daemon and returns the process ID.
```

Reply:
Maybe use a table like in `deployment.rst`